apollo-ios-codegen/Sources/ApolloCodegenLib/Templates/TemplateRenderer.swift

```diff
@@ -7,7 +7,7 @@ enum TemplateTarget: Equatable {
   /// Used in schema types files; enum, input object, union, etc.
   case schemaFile(type: SchemaFileType)
   /// Used in operation files; query, mutation, fragment, etc.
-  case operationFile
+  case operationFile(moduleImports: [String]?)
```

Comment:
Can we make this have a default value of `nil` so we don't need to provide `moduleImports: nil` in all of the unit tests?

I can't remember if you can do default values on enum case associated values? 🤔